### DISCUSSION TITLE
Attacher une variable dans un acteur html.icon

### DISCUSSION THREAD

**Discussion by edejean** [witsa/synapps#577] [Q&A] [answered]
Bonjour, 
Quand on crée une liaison vers une variable dans ce type d'acteur, on perd l'îcone, moyen d'intégrer la liaison dans le contenu html ?
Bien cdt
Eric

**Comment by manufitoussiwit** [comment, marked as answer]
Bonjour,

Si l'on veut ajouter quelque chose au contenu sans casser la partie icone, il faut ajouter un joker : 

![demo ajout joker icon](https://user-images.githubusercontent.com/35595723/228295028-abad5aa5-04b5-4449-97c3-04dd2b79b2c0.gif)

- on ajouter un joker dans le contenu
- on ajoute l'additionnel grâce à la suggestion de Studio
- on peut librement le modifier ou le lié à une donnée.

**Comment by edejean** [comment]
Super merci !!!
C'est vrai que ces jokers existent dans des composites, je n'ai pas imaginé créer un joker directement dans ma vue ;)
